src/components/sidebar-menu/SidebarMenu.styles.ts

```diff
@@ -37,5 +37,14 @@ export const styles = {
         minWidth: 'unset'
       }
     }
+  },
+  listItemContent: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: 1.25
+  },
+  errorIcon: {
+    color: `${palette.error[800]} !important`,
```

Comment:
Do you need `important` here?

Reply:
Yes, the icon inherits title text color otherwise